core/executor/Cargo.toml

```diff
@@ -14,19 +14,19 @@ serializer = { package = "substrate-serializer", path = "../serializer" }
 state_machine = { package = "substrate-state-machine", path = "../state-machine"  }
 runtime_version = { package = "sr-version", path = "../sr-version" }
 panic-handler = { package = "substrate-panic-handler", path = "../panic-handler" }
-wasmi = "0.5.0"
-parity-wasm = "0.31"
-byteorder = "1.3"
-lazy_static = "1.3"
+wasmi = { git = "https://github.com/paritytech/wasmi", branch = "update-parity-wasm" }
```

Comment:
Why do you update to a git version?

Reply:
The latest released version has an old `parity-wasm` dependency, which blocked me from upgrading `parity-wasm`.

Reply:
if parity-wasm has a new release and wasmi rely on the old one maybe we can release a new version of wasmi @pepyakin 
But this can also be done in another PR

Reply:
Wasmi 0.5.1 is out. It should have the updated parity-wasm version.